includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -73,7 +74,14 @@ public function admin_notices() {
 		}
 
 		foreach ( (array) $this->notices as $notice_key => $notice ) {
-			echo '<div class="' . esc_attr( $notice['class'] ) . '" style="position:relative;">';
+			$has_actions = count( $notice['actions'] ) > 0;
+			$div_style   = 'position:relative;';
```

Comment:
Super-tiny nit: I think it would be easier to read if we swapped these two lines so the unconditional set of `$div_style` occurs before we start running any logic related to `$has_actions`.
```suggestion
			$div_style   = 'position:relative;';
			$has_actions = count( $notice['actions'] ) > 0;
```

Reply:
Swapped them in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4669/commits/9ea55bc0a27e8650fb6ae20d13726b16b6064f5e